src/components/idleUsers/card/card.module.scss

```diff
@@ -25,9 +25,13 @@ $theme-pink: #e30464;
   }
 
   .name {
-    margin-top: 3%;
-    margin-bottom: 0;
+    display: flex;
     font-size: 1rem;
     font-weight: 700;
+    width: 160px;
```

Comment:
Do we need width and height?

Reply:
Yes, otherwise overflow won't be possible

Reply:
Yes, otherwise the height and width changes depending upon the length of the name

Reply:
werent we going to use `text-overflow: ellipsis;`

Reply:
There wasn't a requirement of it after using flex because the p tag was not centered so, using flex fixed both of the things